packages/paste-website/src/pages/primitives/combobox-primitive/index.mdx

```diff
@@ -0,0 +1,293 @@
+slug: /primitives/combobox-primitive/
+---
+
+import {graphql} from 'gatsby';
+import {useUID} from 'react-uid';
+import {useComboboxPrimitive} from '@twilio-paste/combobox-primitive';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {ChevronDownIcon} from '@twilio-paste/icons/esm/ChevronDownIcon';
+import {FormLabel, FormInput} from '@twilio-paste/form';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {autocompleteExample, defaultExample} from '../../../component-examples/ComboboxPrimitiveExamples';
+import Changelog from '@twilio-paste/combobox-primitive/CHANGELOG.md';
+
+export const pageQuery = graphql`
+  {
+    allPastePrimitive(filter: {name: {eq: "@twilio-paste/combobox-primitive"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/combobox-primitive/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Combobox Primitive"
+  categoryRoute={SidebarCategoryRoutes.PRIMITIVES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/primitives/combobox"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/primitives-combobox--dropdown-combobox"
+  data={props.data.allPastePrimitive.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+## Guidelines
+
+### About the Combobox Primitive
+
+The combobox primitive is an unstyled functional version of a Combobox. It can be used to build functional and
+accessible comboboxes with or without autocomplete/typeahead features. Our Combobox is built on top of this primitive.
+
+The purpose of providing these unstyled primitives is to cater for instances when the styled Combobox provided
+by Paste, doesn't meet the requirements needed to solve a unique or individual customer problem. At that point
+you are welcome to fallback to this functional primitive to roll your own styled combobox whilst still providing
+a functional and accessible experience to your customers.
+
+This primitive should be used to compose all custom comboboxes to ensure accessibility and upgrade paths.
+
+<Callout variant="warning">
+  <CalloutTitle>Warning</CalloutTitle>
+  <CalloutText>
+    We strongly suggest that all components built on top of this primitive get reviewed by the Design Systems team and
+    goes through the UX Review process to ensure an excellent experience for our customers.
+  </CalloutText>
+</Callout>
+
+## Usage Guide
+
+This package is a wrapper around the [`Downshift` package](https://downshift.netlify.com).
+Our wrapper currently only exposes the useSelect and useCombobox hooks, but renamed for Paste.
+The API's for both hooks are otherwise identical. The reason we chose to just expose the hooks
```

Comment:
We expose the render prop and hook. We need to update this paragraph

Reply:
Should we mention exposing the render prop here? Or add a separate paragraph about using the render prop if not able to use hooks, along with a link out to their docs?

Reply:
I wonder if the future upgrade path would be easier if we don't mention the render props here at all. If there's a lot of demand we can add it in, but removing it might cause more friction down the line than it's worth.

Reply:
Fair, let’s just mention the hook and keep the render prop as an unadvertised back up plan

Reply:
👍